element_test.go

```diff
@@ -815,3 +815,18 @@ func (t T) ElementErrors() {
 	err = el.Context(ctx).Release()
 	t.Err(err)
 }
+
+func (t T) ElementMoveMouseOut() {
+	p := t.page.MustNavigate(t.srcFile("fixtures/click.html"))
+	btn := p.MustElement("button")
+	btn.MustEval(`this.onmouseenter = () => this.dataset['a'] = 1`)
+
+	btn.MustHover()
+	time.Sleep(time.Second * 3)
```

Comment:
why do you need to sleep?

Reply:
The time delay is to see the effect.

Reply:
The program doesn't have eyes, how can the test see it?

Reply:
Aha, in non-headless mode. Then remove it!

Reply:
You should add some code to detect if the movement is as expected, you use the `onmouseout` event to check if the mouse is moved out or not. No need to use any sleep.